tests/e2e/plot-sizing/test_plot_sizing.py

```diff
@@ -52,16 +52,3 @@ def test_output_image_kitchen(page: Page, local_app: ShinyAppProc) -> None:
                 tolerance += 2
             assert abs(rect["width"] - 300) <= tolerance
             assert abs(rect["height"] - 200) <= tolerance
-
-
-def test_decorator_passthrough_size():
```

Comment:
Moved this from e2e test to regular pytest, since it didn't actually use any playwright logic.